Libraries/LibWeb/TrustedTypes/TrustedTypePolicyFactory.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2025, Miguel Sacristán Izcue <miguel_tete17@hotmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <LibJS/Forward.h>
+#include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/Bindings/TrustedTypePolicyFactoryPrototype.h>
+
+namespace Web::TrustedTypes {
+
+class TrustedTypePolicyFactory final : public Bindings::PlatformObject {
+    WEB_PLATFORM_OBJECT(TrustedTypePolicyFactory, Bindings::PlatformObject);
+    GC_DECLARE_ALLOCATOR(TrustedTypePolicyFactory);
+
+public:
+    [[nodiscard]] static GC::Ref<TrustedTypePolicyFactory> create(JS::Realm&);
+
+    virtual ~TrustedTypePolicyFactory() override { }
+
+    // FIXME: Not implemented
+    bool is_html(JS::Value&) const { return false; }
+    bool is_script(JS::Value&) const { return false; }
+    bool is_script_url(JS::Value&) const { return false; }
```

Comment:
It's probably better to mark these methods as `[FIXME]` in the IDL file rather than stubbing them like this - unless something in particular relies on this method being implemented.